commons/src/main/java/org/infinispan/commons/io/UnsignedNumeric.java

```diff
@@ -170,12 +170,14 @@ public static int readUnsignedInt(byte[] bytes, int offset) {
     *
     * @param i int to write
     */
-   public static void writeUnsignedInt(byte[] bytes, int offset, int i) {
+   public static int writeUnsignedInt(byte[] bytes, int offset, int i) {
```

Comment:
:+1: I am surprised we didn't have this before

I wonder if returning the size used may be better, WDYT?